crates/egui/Cargo.toml

```diff
@@ -92,6 +92,7 @@ backtrace = { version = "0.3", optional = true }
 ## Enable this when generating docs.
 document-features = { version = "0.2", optional = true }
 
+bitflags = "2.4"
```

Comment:
This should move up. It is currently under the `#! ### Optional dependencies` heading